lib/features/family/features/profiles/widgets/my_givts_text_button.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:givt_app/core/enums/enums.dart';
+import 'package:givt_app/features/family/app/injection.dart';
+import 'package:givt_app/features/family/extensions/extensions.dart';
+import 'package:givt_app/features/family/features/history/history_cubit/history_cubit.dart';
+import 'package:givt_app/features/family/features/history/history_screen.dart';
+import 'package:givt_app/features/family/shared/widgets/texts/shared_texts.dart';
+import 'package:givt_app/utils/utils.dart';
+
+class MyGivtsTextButton extends StatefulWidget {
```

Comment:
Textbutton is a designsystem element, why don't we do that in code as well? :) then we can force tracking etc like other buttons

Reply:
yes :)